package.json

```diff
@@ -7,7 +7,7 @@
     "android:production": "react-native run-android --no-packager --variant=release",
     "ios": "react-native run-ios",
     "start": "react-native start",
-    "test": "jest --silent --verbose --maxWorkers=1",
+    "test": "jest --verbose --maxWorkers=1",
```

Comment:
Any reason why we had --silent set?

Reply:
I don't know why, so I think it is fine to remove.